esp/templates/program/modules/teachercheckinmodule/missingteachers.html

```diff
@@ -211,6 +272,12 @@ <h1>Missing Teachers for {{ program.niceName }}</h1>
                 </div>
                 <button class="add-flag btn">add flag</button>
             {% endif %}
+            <div class="assignment-name-list fqr-class-assignments">
+                {% for assignment in section.resourceassignments.all %}
+                    {% include "program/modules/resourcemodule/assignment_name.html" %}
+                {% endfor %}
+            </div>
+            <button class="add-assignment btn" data-section="{{ section.id }}" data-geturl="/manage/{{program.getUrlBase}}/getavailableequipment/">checkout resource</button>
```

Comment:
If a teacher was checked in a previous block, then on the checkin page the teacher's box is minimized saying they are already checked in and then the only way to assign them resources for that block (suppose they didn't need an adapter for their first class but need one for their second class) is to uncheck them and then assign the resource and check them in again. So if teachers are checked in previously, we should collapse the checking in part but NOT the resource checkout part. I think?

Reply:
The classes aren't collapsed, the list of sections to be displayed only includes those with at least one unchecked-in teacher. So should we just list all of the sections (ordered as 1) sections with no checked-in teachers, 2) sections with some checked-in teachers, 3) sections with all teachers checked-in) and not have the checked-in teachers box at the bottom?

Reply:
That seems like a good solution!